src/viam/sdk/components/camera/camera.cpp

```diff
@@ -145,7 +126,7 @@ Camera::image_collection Camera::from_proto(viam::component::camera::v1::GetImag
         images.push_back(raw_image);
     }
     image_collection.images = std::move(images);
-    image_collection.captured_at = timestamp_to_time_pt(proto.response_metadata().captured_at());
+    image_collection.metadata = ::viam::sdk::from_proto(proto.response_metadata());
```

Comment:
we're already in the `viam::sdk` namespace here so no need to use it as a prefix.

Reply:
done